### DISCUSSION TITLE
Could we have a latest ruleset_converter tool?

### DISCUSSION THREAD

**Discussion by zhmars** [bromite/bromite#1286] [General]
I'm not sure if it is the right place for this question. But creating an issue is not allowed in the [original repo](https://github.com/bromite/filters).

The pre-built [`ruleset_converter`](https://github.com/bromite/filters/releases/tag/78.0.3904.130) binary is a bit outdated although it still works fine. And there are some updates from upstream since `2019-12-09`. I wonder if there are any improvements for converting a text based filter list.

**Comment by csagan5** [comment]
I have released a new version.

https://github.com/bromite/filters/releases

**Reply by zhmars** [reply]
Thanks a lot!
